### DISCUSSION TITLE
ScriptReference/Unity.Collections.NativeArray_1.Copy

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#585] [Notes]
# ScriptReference/Unity.Collections.NativeArray_1.Copy

https://docs.unity3d.com/ScriptReference/Unity.Collections.NativeArray_1.Copy.html

**Comment by vertxxyz** [comment]
If the `length` is `0` this method will still throw an Exception if any of the other arguments like `dstIndex` are out of range.
